postgis/postgis_after_upgrade.sql

```diff
@@ -205,6 +205,7 @@ SELECT _postgis_drop_function_by_signature('ST_Distance(geography, geography)');
 SELECT _postgis_drop_function_by_signature('ST_Distance(geography, geography, float8, boolean)');
 SELECT _postgis_drop_function_by_signature('ST_Buffer(geometry, float8, cstring)');
 SELECT _postgis_drop_function_by_signature('ST_IsValidDetail(geometry)');
+SELECT _postgis_drop_function_by_signature('ST_RemoveIrrelevantPointsForView(geometry, box2d)');
```

Comment:
Please leave a comment about why we are removing it. See for example the "temporary introduced before x.x.x final". And pass a second argument being the version in which it was deprecated (3.5.0dev  in this case)

Reply:
@strk Comment added.